cpp/ql/src/semmle/code/cpp/models/implementations/Pure.qll

```diff
@@ -23,7 +23,6 @@ class PureStrFunction extends AliasFunction, ArrayFunction, TaintFunction,  Side
         or name = "strnlen"
         or name = "strrchr"
         or name = "strspn"
-        or name = "strstr"
```

Comment:
By _strstr family_ I meant also `strcasestr`, `strchr`, `strchrnul`, and `strrchr`. Instead of enumerating them, you can consider matching them by signature: it's all the functions on this list that return a pointer type.